app/views/spree/admin/product_properties/index.html.haml

```diff
@@ -61,8 +61,9 @@
 
 :javascript
   var properties = #{raw(@properties.to_json)};
-  $("#product_properties input.autocomplete").live("keydown", function(){
-    already_auto_completed = $(this).is('ac_input');
+
+  $("#product_properties").live("keydown", "input.autocomplete", function(){
```

Comment:
Hum, I've staged/tested but still see the error. Maybe we still need to replace the `.live` -> `.on` as on the other file?

Reply:
Yep :see_no_evil: 

Nice catch!

Reply:
I've updated it and force-pushed. It should be fixed now if you restage the new version :+:1